### PR TITLE
Tanh term added

### PR DESCRIPTION
**Pull request by chandlermckane**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Title: '.Tanh()' 
+Description: 'Computes the hyperbolic tangent of a given angle in radians'
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:3]:
```suggestion
Description: 'Computes the hyperbolic tangent of a given angle in radians.'
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.tanh()`** method returns the hyperbolic tangent of a number.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:16]:
```suggestion
The **`Math.tanh()`** method returns the hyperbolic tangent of a given angle in radians.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:17]:
```suggestion
```
Remove extra line break.

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Syntax
+
+```pseudo
+Math.Tahn(x);
+```
+Parameter:This method accepts only single parameters.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:24]:
```suggestion

Parameter: This method accepts only a single parameter.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+`x`:This parameter is the value to be passed to tanh()
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:26]:
```suggestion
`x`: This parameter is the angle in radians.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+`Returns`:This function returns the hyperbolic tangent value of a number.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:27]:
```suggestion

Returns: This function returns the hyperbolic tangent value of `x`.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## Example
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:30]: Can we have an introductory sentence explaining what the example is doing?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@chandlermckane, I did a first review and left a few comments.

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+## Syntax
+
+```pseudo
+Math.Tahn(x);
+```
+
+Parameter: This method accepts only single parameter.
+
+`x`: This parameter is the angle in radians.
+Returns: This function returns the hyperbolic tangent value of `x`.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:27]:
```suggestion

Returns: This function returns the hyperbolic tangent value of `x`.
```
Added line break.

**Review by SSwiniarski — APPROVED** [review]
@chandlermckane Other than an additional line break, this seems good for a second review.

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example
+
+The code is declaring two constants, `x` and `y`, and then calculating the hyperbolic tangent of `x` using the Math.tanh() function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console using console.log().
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:31]:
```suggestion
The code is declaring two constants, `x` and `y`, and then calculating the hyperbolic tangent of `x` using the `Math.tanh()` function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+```cs
+const x = 2;
+const y = Math.tanh(x);
+console.log(y); // Output: 0.9640275800758169
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:36]:
```suggestion
console.log(y);
// Output: 0.9640275800758169
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+`x`: This parameter is the angle in radians.
+Returns: This function returns the hyperbolic tangent value of `x`.
+
+## Example
+
+The code is declaring two constants, `x` and `y`, and then calculating the hyperbolic tangent of `x` using the Math.tanh() function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console using console.log().
+
+```cs
+const x = 2;
+const y = Math.tanh(x);
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:35]:
```suggestion
const y = Math.tanh(x);

```

**Comment by chandlermckane** [comment]
> @chandlermckane It's been two weeks. Are you still working on this entry? Do you need assistance with @yangc95's comments?

I believe I have made the necessary changes.

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+
+```pseudo
+Math.Tahn(x);
+```
+
+This method accepts a single parameter,`x`, which is the angle in radians. It returns the hyperbolic tangent value of `x`.
+
+## Example
+
+The example below declares two constants, `x` and `y`, and then calculates the hyperbolic tangent of `x` using the `Math.tanh()` function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console.
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:28]:
```suggestion
The example below declares two constants, `x` and `y`, and then calculates the hyperbolic tangent of `x` using the `Math.Tanh()` function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console.
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+
+```pseudo
+Math.Tahn(x);
+```
+
+This method accepts a single parameter,`x`, which is the angle in radians. It returns the hyperbolic tangent value of `x`.
+
+## Example
+
+The example below declares two constants, `x` and `y`, and then calculates the hyperbolic tangent of `x` using the `Math.Tanh()` function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console.
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:28]:
```suggestion
The example below declares two double values, `x` and `y`, and then calculates the hyperbolic tangent of `x` using the `Math.Tanh()` function. The resulting value is stored in `y`. Finally, the value of `y` is printed to the console.
```

**Review by yangc95 — APPROVED** [review]
@chandlermckane Please see the changes I've made which include keeping the term consistent with the proper syntax throughout the entry. In addition, I edited some wording and wrote out the code to match standard C# syntax.

Please let me know if you have any concerns. Looks good for merge now!

**Comment by yangc95** [comment]
@chandlermckane here's the live link: https://www.codecademy.com/resources/docs/c-sharp/math-functions/tanh

Screenshot 2023-04-10 at 6 49 25 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> 
> 
> Make sure to also set your visibility to “everyone” under privacy.
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.
> 
> In addition, we may reach out in the future if any adjustments to these entries are required.
>

cc: @SSwiniarski

**Comment by chandlermckane** [comment]
I have updated my profile and linked my Github. How long will it take to populate me as a contributor?

**Pull request merged**